src/webview/js/rewriteHtml.js

```diff
@@ -86,6 +86,26 @@ const rewriteTime = (element: Element | Document) => {
   });
 };
 
+/**
+ * Add missing elements to spoiler blocks under the given root, inclusive.
+ *
+ * 1. Adds 'Spoiler' header if missing.
+ * 2. Adds the spoiler expanded arrow indicator. The arrow is rendered using CSS.
+ */
+const rewriteSpoilers = (element: Element | Document) => {
+  const spoilerHeaders: NodeList<HTMLElement> = element.querySelectorAll('div.spoiler-header');
+  spoilerHeaders.forEach(e => {
+    const toggle_button_html =
+      '<span class="spoiler-button" aria-expanded="false"><span class="spoiler-arrow"></span></span>';
```

Comment:
Hmm, I think this calls for a tweak to the comment on `toggleSpoilers`. The `.spoiler-button` and its child the `.spoiler-arrow` don't actually come from the server -- they're added by our own code, here.

In particular that means that if for some reason we wanted that part to be different, we could change it right within the app.